scripts/build_test.py

```diff
@@ -107,9 +107,18 @@ def test_minify_and_create_sourcemap(self) -> None:
 
     def test_minify_and_create_sourcemap_under_docker_environment(self) -> None:
         """Tests _minify_and_create_sourcemap with an invalid filepath."""
-        with self.swap(feconf, 'OPPIA_IS_DOCKERIZED', True):
-            build._minify_and_create_sourcemap(  # pylint: disable=protected-access
-                INVALID_INPUT_FILEPATH, INVALID_OUTPUT_FILEPATH)
+        with self.assertRaisesRegex(
+            subprocess.CalledProcessError,
+            'returned non-zero exit status 1') as called_process:
```

Comment:
yes this is expected because we are setting the `OPPIA_IS_DOCKERIZED` env as true, and then it tries to find the file paths as `/app/oppia` which is for the docker container env path

Reply:
Earlier it wasn't throwing error and now it does, right? Do we need to change the script/functionality to avoid throwing errors? (Maybe I'm not understanding the workflow clearly here, can you please help me?)

Reply:
yeah, the reason being is earlier the prod mode was not working under Docker. this PR also fixes that. we need to change the command little bit which is a part of that fix

FYI, I added this test earlier in M1

Reply:
As per discussion, @Shivkant-Chauhan is going to resolve it!

Reply:
is this resolved?